phpBB/posting.php

```diff
@@ -43,6 +42,7 @@
 $cancel		= (isset($_POST['cancel']) && !isset($_POST['save'])) ? true : false;
 
 $refresh	= (isset($_POST['add_file']) || isset($_POST['delete_file']) || isset($_POST['cancel_unglobalise']) || $save || $load || $preview);
+$submit		= $request->is_set_post('post') && !$refresh;
```

Comment:
Well, if `$preview` is true, `$refresh` will be true too. But I guess it wouldn't hurt to change it to check `$preview` as well here. I think it may improve the readability, so... yeah.. :D